clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ActiveActiveStoreIngestionTask.java

```diff
@@ -438,6 +438,10 @@ protected void processMessageAndMaybeProduceToKafka(
           storeName,
           versionNumber,
           LatencyUtils.getLatencyInMS(beforeProcessingRecordTimestamp));
+
+      // Record the last ignored offset
+      partitionConsumptionState
+          .updateLatestIgnoredUpstreamRTOffset(kafkaClusterIdToUrlMap.get(kafkaClusterId), sourceOffset);
```

Comment:
I think we need to update ignored here
 https://github.com/linkedin/venice/blob/main/clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java#L910 as well?

Reply:
But that part of logic should be just sanity check and checking if the message is control msg?

Reply:
Yeah, but they should still be considered in the lag calculation, right?

Reply:
It's a good question.  Control messages aren't sent by the user so I think it's debatable if they should factor into the calculation since acceptable lag is set by the user and that user is making a call on the data that they are writing.  Though I doubt one message is going to break the bank.

I'm inclined to include if only because I'm scared of off by one race conditions.  It's probably more vexing in the systems behavior to lend more special dispensation to message types when it's not needed.